bitstreams.md

```diff
@@ -115,6 +116,18 @@ The supported **Request Headers** are:
 * Range: not implemented yet. Provide support to partial content download
 * If-None-Match: not implemented yet. Support for cache control
 
+### Main Thumbnail
+**/api/core/bitstreams/<:uuid>/thumbnail**
+
+It returns the bitstream which represents the thumbnail of the bitstream
```

Comment:
Since Thumbnails are also Bitstreams, what should happen if you call `/thumbnail` on a Bitstream in the `THUMBNAIL` Bundle? 

Overall, the approach here seems reasonable to me, I'm just trying to better understand the proposed design, so that we can enhance the REST Contract perhaps.  

My assumption is that likely this endpoint returns a 204 when called for any Bitstream that is *not* in the `ORIGINAL` bundle.  If that's true, we may want to call that out in the Contract by saying something like:

> Returns the bitstream which represents the thumbnail of the specified bitstream. At this time, thumbnails are only supported for bitstreams in the `ORIGINAL` bundle.

Reply:
That's correct, I've added this info to the contract